CanvasColorSpaceProposal.md

```diff
@@ -89,6 +87,7 @@ canvas.getContext('2d', { colorSpace: "rec2020",
 The ``colorSpace`` attribute specifies the color space for the backing storage of the canvas.
 * Color spaces match their respective counterparts as defined in the [predefined color spaces](https://www.w3.org/TR/css-color-4/#predefined) section of the [CSS Color Module Level 4](https://www.w3.org/TR/css-color-4) specification.
 * When an unsupported color space is requested, the color space shall fall back to ``"srgb"``.
+* Implementations should not limit the set of exposed color spaces based on the capabilities of the display.
```

Comment:
The pull request comment "The color gamut media queries exist to make the determination about which color space will max-out the fidelity of a given monitor." is directly related to this statement. Could something similar be added as a second sentence of this bullet point, or perhaps made into non-normative text?

Reply:
Good point -- added a sentence to the bullet point clarifying this.